ethereum/p2p/build.gradle

```diff
@@ -43,8 +43,12 @@ dependencies {
   implementation 'org.apache.logging.log4j:log4j-api'
   implementation 'org.apache.tuweni:tuweni-bytes'
   implementation 'org.apache.tuweni:tuweni-crypto'
-  implementation 'org.apache.tuweni:tuweni-devp2p'
```

Comment:
This change seems to be independent, should it go in its own PR?

Reply:
It's a good question. I considered removing it, but since it solves for an existing open issue, i left it in and instead linked this PR to that issue. I think this falls under "clean as you go" but I admit that can be a subjective decision.